generators/blank-marketplace/templates/package.json

```diff
@@ -0,0 +1,113 @@
+    "build:dev": "webpack",
+    "prebuild:prod": "npm stop; npm run clean; npm run check-node-version",
+    "build:prod": "NODE_ENV=production webpack -p",
+    "deploy": "marketplace-kit deploy -e $npm_package_config_target",
+    "deploy:staging": "npm run build:prod; npm run deploy --<%= PROJECT_NAME %>:target=staging",
+    "deploy:prod": "npm run build:prod; npm run deploy --<%= PROJECT_NAME %>:target=prod",
+    "test": "rm -rf test/e2e/screenshots/*; node nightwatch-local.conf.js; nightwatch --config nightwatch-local.conf.js",
+    "test:browserstack": "nightwatch --config nightwatch-browserstack.conf.js --env chrome",
+    "test:browserstack:local": "node scripts/bs-local-runner.js -c nightwatch-browserstack-local.conf.js # doesnt work yet",
+    "test:one": "rm -rf test/e2e/screenshots/*; node nightwatch-local.conf.js; nightwatch --config nightwatch-local.conf.js --test $1"
+  },
+  "pre-commit": {
+    "silent": true,
+    "run": [
+      "lint"
+    ]
+  },
+  "lint-staged": {
+    "concurrent": true,
+    "subTaskConcurrency": 3,
+    "linters": {
+      "**/*.scss": [
+        "lint:fix",
+        "git add"
+      ],
+      "{src,test}/**/*.js": [
+        "lint:fix",
+        "git add"
+      ],
+      "**/*.graphql": [
+        "lint:fix",
+        "git add"
+      ]
+    }
+  },
+  "browserslist": [
+    "last 2 versions",
+    "iOS > 7",
+    "Safari > 6"
+  ],
+  "license": "MIT",
+  "private": true,
+  "devDependencies": {
+    "autoprefixer": "^7.2.1",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-plugin-lodash": "^3.3.2",
+    "babel-plugin-transform-object-assign": "^6.22.0",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
+    "browserstack-local": "^1.3.0",
+    "cache-loader": "^1.2.0",
+    "check-node-version": "^3.1.1",
+    "css-loader": "^0.28.7",
+    "css-mqpacker": "^6.0.1",
+    "dotenv": "^4.0.0",
+    "dynamic-public-path-webpack-plugin": "^1.0.3",
+    "exports-loader": "^0.6.4",
+    "expose-loader": "^0.7.4",
+    "extract-text-webpack-plugin": "^3.0.2",
+    "file-loader": "^1.1.5",
+    "glob-all": "^3.1.0",
+    "jquery": "^3.2.1",
+    "lint-staged": "^6.0.0",
+    "lodash-webpack-plugin": "^0.11.4",
+    "nightwatch": "^0.9.19",
+    "nightwatch-commands": "^3.0.0",
+    "node-sass": "4.7.2",
+    "postcss": "^6.0.14",
+    "postcss-calc": "^6.0.1",
+    "postcss-devtools": "^1.1.1",
+    "postcss-fixes": "^2.0.1",
+    "postcss-focus": "^3.0.0",
+    "postcss-loader": "^2.0.9",
+    "postcss-pseudoelements": "^5.0.0",
+    "pre-commit": "^1.2.2",
+    "prettier-eslint-cli": "^4.4.2",
+    "sass-loader": "^6.0.6",
+    "script-loader": "^0.7.2",
+    "selenium-download": "^2.0.10",
+    "style-loader": "^0.19.0",
+    "uglifyjs-webpack-plugin": "^1.1.2",
+    "webpack": "^3.10.0",
+    "webpack-require-from": "^1.0.1",
+    "write-file-webpack-plugin": "^4.2.0"
+  },
+  "dependencies": {
+    "axios": "^0.17.1",
```

Comment:
This needs to be cleaned up also - since blank project shouldnt have already dependencies like lodash/cropper